layers/+tools/shell/README.org

```diff
@@ -221,12 +223,26 @@ in the current buffer instead of a popup.
 
 ** Multi-term
 
-| Key binding | Description                  |
-|-------------+------------------------------|
-| ~SPC m c~   | create a new multi-term      |
-| ~SPC m n~   | go to next multi-term        |
-| ~SPC m p~   | go to previous multi-term    |
-| ~SPC p $ t~ | run multi-term shell in root |
+
+*** Hybrid & Vim mode
```

Comment:
Remove the separate modes, just list the bindings individually with these two next to each other:
```
| ~SPC m N~   | go to previous multi-term      |
| ~SPC m p~   | go to previous multi-term      |
```

Under the `** Multi-term` heading.